WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt

```diff
@@ -132,32 +132,37 @@ title_sigecempa = {
 }
 
 title_court_celtic_poet = { # Celtic Poet/Bard
-	dignity = 0.25
+	dignity = 1
 	realm_in_name = yes
 	grant_limit = 1
 	opinion_effect = 30
 	
 	monthly_salary = 0.02
-	monthly_prestige = 0.01
+	monthly_prestige = 0.1
```

Comment:
No, because burden of proof lies on you. If you can't provide it you loose the argument, you don't convince for the utility of this and the change isn't implemented.